mlptrain/configurations/configuration_set.py

```diff
@@ -24,8 +25,12 @@ def __init__(self,
         Arguments:
             args: Either strings of existing files (e.g. data.npz) or
                   individual configurations.
+
+            allow_duplicates: Should duplicate configurations be supported? For
```

Comment:
Could there be problems if there are duplicates in an US trajectory, it would change the number of configs per window which might mess up the counting along the line?

Reply:
if anything `allow_duplicates = True` is the desired behaviour then?